Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2024,11 +2042,8 @@ extension BrowserViewController: UIAlertViewDelegate {
 
     func alertView(alertView: UIAlertView, clickedButtonAtIndex buttonIndex: Int) {
         if buttonIndex == CrashPromptIndex.Restore.rawValue {
-            tabManager.restoreTabs()
-        }
-
-        // In case restore fails, launch at least one tab
-        if tabManager.count == 0 {
```

Comment:
Do we want to remove this check? The `self.restoreTabs()` below could still fail, so it seems like it'd be safer to leave this in.